packages/fast-tooling-react/src/css-editor/border/border.props.ts

```diff
@@ -0,0 +1,48 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { CSSBorderClassNameContract } from "./border.style";
+
+export enum BorderStyleValue {
+    dashed = "dashed",
+    dotted = "dotted",
+    double = "double",
+    groove = "groove",
+    hidden = "hidden",
+    inherit = "inherit",
+    initial = "initial",
+    inset = "inset",
+    none = "none",
+    outset = "outset",
+    ridge = "ridge",
+    solid = "solid",
+    unset = "unset",
+}
+
+export interface CSSBorderState {
+    borderColor?: string;
+    borderStyle?: BorderStyleValue;
+    borderWidth?: string;
+}
+
+export interface CSSBorderValues {
+    border?: string;
+    borderColor?: string;
+    borderStyle?: BorderStyleValue;
+    borderWidth?: string;
```

Comment:
This suggests that we might return any subset of these values, but the reality is that we only ever return border, I think if that's what we expect you can remove `borderColor`, `borderStyle`, `borderWidth`